client/state/posts/likes/reducer.js

```diff
@@ -80,7 +92,7 @@ export const itemReducer = withSchemaValidation(
 
 				return {
 					likes,
-					iLike: state.iLike,
+					iLike: true,
```

Comment:
@roo2, you're 100% right that I introduced a regression. I am 99% sure it's due to this line, but I will keep digging.